zebra/dplane_fpm_nl.c

```diff
@@ -1260,18 +1265,18 @@ static int fpm_process_queue(struct thread *t)
 		if (ctx == NULL)
 			break;
 
-		fpm_nl_enqueue(fnc, ctx);
+		if (fpm_nl_enqueue(fnc, ctx) == -1)
+			atomic_fetch_add_explicit(
+				&fnc->counters.dplane_contexts_failed, 1,
+				memory_order_relaxed);
 
 		/* Account the processed entries. */
 		atomic_fetch_add_explicit(&fnc->counters.dplane_contexts, 1,
 					  memory_order_relaxed);
 		atomic_fetch_sub_explicit(&fnc->counters.ctxqueue_len, 1,
 					  memory_order_relaxed);
 
-		dplane_ctx_set_status(ctx, ZEBRA_DPLANE_REQUEST_SUCCESS);
-		dplane_provider_enqueue_out_ctx(fnc->prov, ctx);
```

Comment:
Aren't these steps still needed - how will the context object get back to the dplane framework?

Reply:
did not mean to remove this.  Fixing